Bmx.CommandLine/CommandLine.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
```

Comment:
This seems like a really big class - is there an opportunity to split this into smaller classes (only if it makes sense)?

Reply:
Oh, I now see that they are all private methods. This is probably fine.